docs/userguide/campaigns.md

```diff
@@ -1,25 +1,156 @@
+[`Campaign.from_config`](baybe.campaign.Campaign.from_config).
+As fully specifying a configuration takes too much, we refer to the corresponding
+[example](./../../examples/Serialization/create_from_config).
+
+## Getting recommendations
+
+### Basics
+
+```{attention}
+Adding recommendations and measurements using the `recommend` and `add_measurements`
+functions is the safe only way to inform a `Campaign` object about new measurements.
+These functions update the necessary metadata that is crucial for the proper
+execution of a campaign. It is important to rely on these functions to maintain the
+integrity and reliability of the campaign's execution.
+```
+
+To obtain a recommendation for the next experiment, we can query the campaign and use
+the [`recommend`](baybe.campaign.Campaign.recommend) function. The function takes only
+one argument, which is the `batch_quantity` keyword. This specifies the desired size of
+the batch of experiments to be conducted.
+
+```python
+rec = campaign.recommend(batch_quantity=3)
+```
+
+The `recommend` function returns a `DataFrame` with `batch_quantity` many rows, each 
+representing a set of parameters from the search space.
+
+### Influence of the batch size
+
+Regarding the choice of a suitable `batch_size`, it is important to understand the
+difference between performing multiple recommendations with batch size of 1 and a single
+recommendation with a larger batch size.
+* **Larger batch size**: When using a larger batch size, the recommended experiments are
+chosen to *jointly* optimized the acquisition function.
+This means that the recommendations are made considering the interaction of multiple
+experiments together.
+* **Smaller batch size**: When making several smaller recommendations, each *individual*
+recommendation optimizes the acquisition function at the specific point in time when it
+is requested. In this case, the recommendations are made independently of each other
+without considering the joint optimization.
+
+### Caching of recommendations
+
+Whenever recommendations are made, the `Campaign` object caches them. If measurements
+for the recommendations are added, then the cached recommendations are deleted. However,
+if no measurements are added and the `recommend` function is called again, then the
+`Campaign` object simply returns the cached recommendations instead of generating new
+ones. This caching mechanism helps to optimize performance by avoiding unnecessary
+re-computations when measurements are not provided.
+
+## Adding measurements
+
+To add measurements, we expand the  `DataFrame` that was created by the `recommend`
+function by adding a new column for the target. We can then provide the campaign with
+these measurements and receive a new recommendation.
+
+```python
+rec["quality"] = [2,4,9]
+campaign.add_measurements(rec)
+new_rec = campaign.recommend(batch_quantity=5)
+```
+
+In discrete or hybrid search space, numerical measurements are required to fall into a
```

Comment:
should probably be rephrased to something like `For discrete parameters, measurements are required...` to be accurate (eg in hybrid space the conti params do not need to be in tolerance)